Foundation/NSCoder.swift

```diff
@@ -253,13 +253,34 @@ public class NSCoder : NSObject {
         NSUnimplemented()
     }
     
-    public var allowedClasses: Set<NSObject>? {
+    public var allowedClasses: [AnyClass]? {
         get {
             NSUnimplemented()
         }
     }
     
     public func failWithError(error: NSError) {
-        NSUnimplemented()
+        if let debugDescription = error.userInfo["NSDebugDescription"] {
```

Comment:
this is incorrect; failWithError is a method to halt the decoding of the archive such that the top level object will then return nil from the decoding; failure is not fatal on darwin it just is an error